package.json

```diff
@@ -25,17 +25,20 @@
   },
   "types": "dist/lib/index.d.ts",
   "scripts": {
-    "build": "npm run build:assets && npm run build:css && npm run build:lib && npm run build:client",
+    "build": "npm run build:assets && npm run build:css && npm run build:lib && npm run build:client && npm run build:client:react",
     "prebuild:assets": "shx rm -rf dist/assets",
     "build:assets": "ts-node scripts/copy-govuk-static-assets.ts",
     "prebuild:client": "shx rm -rf dist/client",
     "build:client": "npm run build:client:umd && npm run build:client:es && tsc --project src/client",
     "build:client:umd": "webpack --config webpack.config.ts",
     "build:client:es": "babel src/client/index.ts --extensions .ts,.tsx --out-file dist/client/moduk-frontend.mjs",
+    "build:client:react": "babel src/react/ --extensions .ts,.tsx --out-dir dist/react/",
```

Comment:
Just noticed that the transpiled files still contain JSX. We probably need to add the preset `@babel/preset-react` to `babel.config.mjs` (could use [`env`](https://babeljs.io/docs/options#env) and `BABEL_ENV` to restrict it to the React build).